src/components/tooltip/README.md

```diff
@@ -170,37 +170,37 @@ override the `pointer-events` on the disabled element.
+| `noninteractive`     | `false`          | Wether the tooltip should not be user-interactive                                                                                                                                                          | `true` or `false`                                                                                                                                |
 | `variant`            | `null`           | Contextual color variant for the tooltip                                                                                                                                                                   | Any contextual theme color variant name                                                                                                          |
 | `custom-class`       | `null`           | A custom classname to apply to the tooltip outer wrapper element                                                                                                                                           | A string                                                                                                                                         |
-| `id`                 | `null`           | An ID to use on the tooltip root element. If none is provided, one will automatically be generated. If you do provide an ID, it _must_ be guaranteed to be unique on the rendered page.                    | A valid unique element ID string                                                                                                                 |
+| `id`                 | `null`           | An ID to use on the tooltip root element. If none is provided, one will automatically be generated. If you do provide an ID, it _must_ be guaranteed to be unique on the rendered page                     | A valid unique element ID string                                                                                                                 |
 
-### Interactive tooltips
+### Noninteractive tooltips
 
-BootstrapVue's tooltips can be made user-interactive by setting the `interactive` prop to `true`.
-This option is disabled by default to match Bootstraps default behavior:
+BootstrapVue's tooltips are user-interactive by default for accessability reasons. To restore
+Bootstraps default behavior apply the `noninteractive` prop:
 
 ```html
 <div class="text-center">
   <div>
     <b-button id="tooltip-button-interactive">My tooltip is interactive</b-button>
-    <b-tooltip target="tooltip-button-interactive" interactive>I will stay open</b-tooltip>
+    <b-tooltip target="tooltip-button-interactive">I will stay open</b-tooltip>
```

Comment:
Maybe `I will stay open when hovered` to be a bit more clear.

Reply:
And since it is no longer a breaking change, maybe we could add it to release 2.2.0 (would require updating the docs/meta to say added in 2.2.0)